examples/06_form-demo/src/components/funcs.ts

```diff
@@ -0,0 +1,17 @@
+'use server';
+
+import { unstable_rerenderRoute } from 'waku/router/server';
+
+// module state on server
+let message = '';
+
+export const getMessage = async () => message;
+
+export const greet = async (formData: FormData) => {
+  // simulate a slow server response
+  await new Promise((resolve) => setTimeout(resolve, 1000));
+  message = `Hello ${formData.get('name') || 'Anonymous'} from server!`;
+  unstable_rerenderRoute('/');
```

Comment:
It's nice to have this in this example, but it makes me wonder if this example is more than just "form demo".

Reply:
What would you rename it to? 🤔

Reply:
No immediate ideas. 😅

Reply:
"form demo" seems fitting to me I think. `rerenderRoute` can be used for other things, but progressive forms will be a very normal use of it.